test/compilable/minimal_lib.d

```diff
@@ -0,0 +1,9 @@
+// REQUIRED_ARGS: -lib
+
+// This test ensures that a library created in D that does not rely on any
+// runtime features can be compiled without object.d
```

Comment:
How does this test ensures what's described here?

Reply:
Actually, I had an error in the test:  `-conf=` must be added to avoid importing object.d from the default runtime.  Therefore, there is no object.d in the import path.  The fact that this test compiles with the `-conf=` and `-lib` flags is enough to ensure it the test passes.  Go ahead and try it for yourself locally without this change, and you will find it doesn't compile.  However, after apply this PR, it will.

Reply:
The compiler will not try to find a config globally somewhere in `/etc` with the `-conf=` flag?

Reply:
That's correct.  `-conf=` basically means "don't use any .conf file".  Because there is no .conf file being used the are no import directories like `-I/usr/include`.  Therefore the `object.d` file that typically ships with DMD is not imported.